libvirt/tests/src/virtual_disks/virtual_disks_nvme.py

```
@@ -147,8 +203,118 @@ def create_customized_disk(params, pci_address):
+        LOG.debug("Original disk source: %s", original_disk_source)
+        if vm.is_dead():
+            vm.start()
+        LOG.debug("The current guest xml is: %s" % (virsh.dumpxml(vm_name).stdout_text))
+        vm.wait_for_login().close()
+
+        LOG.info("TEST_STEP2: Execute blockcopy command with option '%s'.", specific_option)
+        if with_luks:
+            cmd = ("qemu-img create -f qcow2 %s %s %s" % (luks_extra_parameter,
+                                                         nvme_partition_path,
+                                                         disk_size_bytes))
+            process.run(cmd, shell=True, ignore_status=False)
```

Ruff (0.14.4)

Function call with `shell=True` parameter identified, security issue

(S604)